app/src/androidTest/java/com/litewallet/analytics/ConstantsTests.java

```diff
@@ -15,6 +15,7 @@
 
 import java.net.URI;
 
+@Deprecated
```

Comment:
@andhikayuana  Trying to run this on a new AS on a new test Mac and it cannot access the test module. 
How do we make sure it is reproducable of all Litewallet devs?

Reply:
noted, I'll create a document for this